tests/operators/test_agnostic_aggregate_check.py

```diff
@@ -33,253 +36,213 @@
 
 import astro.sql as aql
 from astro.sql.table import Table
-from tests.operators.utils import DEFAULT_SCHEMA
+from tests.operators.utils import DEFAULT_SCHEMA, get_dag, get_table_name, run_dag
 
 log = logging.getLogger(__name__)
 DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+CWD = pathlib.Path(__file__).parent
+
+DATABASES_LIST = ["postgres", "bigquery", "sqlite", "snowflake"]
```

Comment:
I really like this constant!
This list could live somewhere else - so we can reuse it in multiple tests. What do you think of placing it in `constants.py`, and naming it `SUPPORTED_DATABASES`?

Reply:
Done, make sense.